speech/grpc/transcribe_streaming.py

```diff
@@ -126,25 +174,34 @@ def listen_print_loop(recognize_stream):
 
         # Exit recognition if any of the transcribed phrases could be
         # one of our keywords.
-        if any(re.search(r'\b(exit|quit)\b', alt.transcript)
+        if any(re.search(r'\b(exit|quit)\b', alt.transcript, re.I)
                for result in resp.results
                for alt in result.alternatives):
             print('Exiting..')
```

Comment:
Ah - probably one of the threads isn't exiting correctly -_-;
Just to make sure - you're sync'ed to the latest changes, right?